src/com/owncloud/android/lib/common/OwnCloudClient.java

```diff
@@ -54,135 +55,135 @@
-    		if (proxyPortSt != null && proxyPortSt.length() > 0) {
-    			proxyPort = Integer.parseInt(proxyPortSt);
-    		}
-    	} catch (Exception e) {
-    		// nothing to do here
-    	}
-
-    	if (proxyHost != null && proxyHost.length() > 0) {
-	    	HostConfiguration hostCfg = getHostConfiguration();
-	    	hostCfg.setProxy(proxyHost, proxyPort);
-	    	Log_OC.d(TAG, "Proxy settings: " + proxyHost + ":" + proxyPort);
-    	}
-	}
-
-
-	public void setCredentials(OwnCloudCredentials credentials) {
-    	if (credentials != null) {
-    		mCredentials = credentials;
-    		mCredentials.applyTo(this);
-    	} else {
-    		clearCredentials();
-    	}
+        String proxyHost = System.getProperty("http.proxyHost");
+        String proxyPortSt = System.getProperty("http.proxyPort");
+        int proxyPort = 0;
+        try {
+            if (proxyPortSt != null && proxyPortSt.length() > 0) {
+                proxyPort = Integer.parseInt(proxyPortSt);
+            }
+        } catch (Exception e) {
+            // nothing to do here
+        }
+
+        if (proxyHost != null && proxyHost.length() > 0) {
+            HostConfiguration hostCfg = getHostConfiguration();
+            hostCfg.setProxy(proxyHost, proxyPort);
+            Log_OC.d(TAG, "Proxy settings: " + proxyHost + ":" + proxyPort);
+        }
+    }
+
+
+    public void setCredentials(OwnCloudCredentials credentials) {
+        if (credentials != null) {
+            mCredentials = credentials;
+            mCredentials.applyTo(this);
+        } else {
+            clearCredentials();
+        }
     }
-    
+
     public void clearCredentials() {
-		if (!(mCredentials instanceof OwnCloudAnonymousCredentials)) {
-			mCredentials = OwnCloudCredentialsFactory.getAnonymousCredentials();
-		}
-		mCredentials.applyTo(this);
-	}
-    
+        if (!(mCredentials instanceof OwnCloudAnonymousCredentials)) {
+            mCredentials = OwnCloudCredentialsFactory.getAnonymousCredentials();
+        }
+        mCredentials.applyTo(this);
+    }
+
     /**
      * Check if a file exists in the OC server
-     * 
-     * @deprecated	Use ExistenceCheckOperation instead
-     * 
-     * @return              	'true' if the file exists; 'false' it doesn't exist
-     * @throws  	Exception   When the existence could not be determined
+     *
+     * @return 'true' if the file exists; 'false' it doesn't exist
+     * @throws Exception When the existence could not be determined
+     * @deprecated Use ExistenceCheckOperation instead
      */
     @Deprecated
     public boolean existsFile(String path) throws IOException, HttpException {
         HeadMethod head = new HeadMethod(getWebdavUri() + WebdavUtils.encodePath(path));
         try {
             int status = executeMethod(head);
             Log_OC.d(TAG, "HEAD to " + path + " finished with HTTP status " + status +
-            		((status != HttpStatus.SC_OK)?"(FAIL)":""));
+                ((status != HttpStatus.SC_OK) ? "(FAIL)" : ""));
```

Comment:
This log is too long and difficult to read, you could set ((status != HttpStatus.SC_OK) ? "(FAIL)" : "")) into a variable before the logging.